app/controllers/articles.server.controller.js

```diff
@@ -12,8 +12,17 @@ var mongoose = require('mongoose'),
  * Create a article
  */
 exports.create = function(req, res) {
+
+	// Filters out control fields
+	delete req.body.updatedAt;
+	delete req.body.updatedBy;
+	delete req.body.createdAt;
+	delete req.body.createdBy;
+
 	var article = new Article(req.body);
-	article.user = req.user;
+
+	article.createdBy = req.user;
```

Comment:
Change this:
```
delete req.body.updatedAt;
delete req.body.updatedBy;
delete req.body.createdAt;
delete req.body.createdBy;
var article = new Article(req.body);
article.user = req.user;
article.createdBy = req.user;
```
to this:
```
delete req.body.updatedAt;
delete req.body.updatedBy;
delete req.body.createdAt;
article.createdBy = req.user;
article.user = req.user;
var article = new Article(req.body);
```
Doesn't it make more sense to not delete then set, and also why not assign and set up everything first, then just save the new article, don't modify it.